gms/feature_service.hh

```diff
@@ -124,6 +124,7 @@ public:
     gms::feature aggregate_storage_options { *this, "AGGREGATE_STORAGE_OPTIONS"sv };
     gms::feature collection_indexing { *this, "COLLECTION_INDEXING"sv };
     gms::feature large_collection_detection { *this, "LARGE_COLLECTION_DETECTION"sv };
+    gms::feature deleted_rows_detection { *this, "DELETED_ROWS_DETECTION"sv };
```

Comment:
This name is too generic, a better name would be `LARGE_PARTITION_DETECTOR_DEAD_ROWS_DETECTION`.

Reply:
This change adds cluster feature protection for two new counters in `system.large_partitions`: `dead_rows` and `range_tombstones`. So I have renamed the cluster feature to `RANGE_TOMBSTONE_AND_DEAD_ROWS_DETECTION`.